AutocompleteClient/FW/Logic/Request/CIOQuizzesQuery.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  CIOQuizzesQuery.swift
+//  Constructor.io
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+/**
+ Struct encapsulating the necessary and additional parameters required to execute a search query.
+ */
+public struct CIOQuizzesQuery: CIORequestData {
+    /**
+     The id of the quiz
+     */
+    public let quizId: String
+
+    /**
+     A list of answers in the format ?a=[option_id_1]&a=[option_id_2] with one a parameter for each question. The expected format for each parameter depends on the question type, as described below
```

Comment:
Similar to the other file, I think we should write a different description than using the one from the docs and link out to the documentation. Something like:

```
A list of answers that be in the following formats:
* Single Select: "answer1"
* Multiple Select: "answer1, answer2"
* Open Text: "true"
* Cover Page: "seen"

Please refer to <insert link to docs> for additional details
```

Lmk what you think